pkg/grafana/alertgroup-handler.go

```diff
@@ -212,18 +212,52 @@ func (h *AlertRuleGroupHandler) updateAlertRule(rule *models.ProvisionedAlertRul
 	return err
 }
 
-func (h *AlertRuleGroupHandler) putAlertRuleGroup(resource grizzly.Resource) error {
-	// TODO: Turn spec into a real models.AlertRuleGroup object
+func unMarshalAlertRuleGroup(resource grizzly.Resource) (*models.AlertRuleGroup, error) {
 	data, err := json.Marshal(resource.Spec())
 	if err != nil {
-		return err
+		return nil, err
 	}
 	var group models.AlertRuleGroup
 	err = json.Unmarshal(data, &group)
 	if err != nil {
-		return err
+		return nil, err
+	}
+	return &group, nil
+}
+
+func updateResourceToHaveUids(existing, resource grizzly.Resource) (*models.AlertRuleGroup, error) {
+	group, err := unMarshalAlertRuleGroup(existing)
```

Comment:
```suggestion
	existingGroup, err := unMarshalAlertRuleGroup(existing)
```

IMHO it's better to have the different groups as separate variables to keep things clear

Reply:
Done